src/quill-editor.component.ts

```diff
@@ -105,7 +105,8 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
       placeholder: placeholder,
       readOnly: this.readOnly || false,
       theme: this.theme || 'snow',
-      formats: this.formats
+      formats: this.formats,
+      bounds: this.bounds || document.body
     });
 
```

Comment:
haha lols, i thought i fixed this long time ago. But i accidentally added it to line 127 in the .emit data of selectionChanged. Could you remove it there, Please?